source/common/event/dispatcher_impl.h

```diff
@@ -67,6 +68,9 @@ class DispatcherImpl : Logger::Loggable<Logger::Id::main>,
   Network::UdpListenerPtr createUdpListener(Network::SocketSharedPtr socket,
                                             Network::UdpListenerCallbacks& cb) override;
   TimerPtr createTimer(TimerCb cb) override;
+  TimerPtr createScaledTimer(ScaledRangeTimerManager::TimerType timer_type, TimerCb cb) override;
+  TimerPtr createScaledTimer(ScaledTimerMinimum minimum, TimerCb cb) override;
```

Comment:
Would be good to have some basic tests for these methods in dispatcher_impl_test